scripts/eval/eval-prompt-change.py

```
@@ -0,0 +1,561 @@
+
+# ---------------------------------------------------------------------------
+# Prompt loading (before/after)
+# ---------------------------------------------------------------------------
+
+def load_prompt_from_ref(prompt_path: str, ref: str) -> str:
+    """Load prompt text from a git ref (branch, commit, tag).
+
+    Raises RuntimeError if the file does not exist at that ref.
+    """
+    try:
+        result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)